Please get [failure logs here](https://dev.azure.com/vcpkg/c1ee48cb-0df2-4ab3-8384-b1df5a79fe53/_apis/build/builds/96139/artifacts?artifactName=failure%20logs%20for%20x64-osx&api-version=7.1&%24format=zip):
```
Run Build Command(s): /usr/local/bin/ninja -v -v -j13 install
[1/4] /Library/Developer/CommandLineTools/usr/bin/c++ -DFASTGLTF_ENABLE_DEPRECATED_EXT=0 -DFASTGLTF_USE_CUSTOM_SMALLVECTOR=0 -DSIMDJSON_THREADS_ENABLED=1 -I/Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/include -isystem /Users/vagrant/Data/installed/x64-osx/include -fPIC -g -std=gnu++17 -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk -MD -MT CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o -MF CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o.d -o CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o -c /Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/src/fastgltf.cpp
FAILED: CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o 
/Library/Developer/CommandLineTools/usr/bin/c++ -DFASTGLTF_ENABLE_DEPRECATED_EXT=0 -DFASTGLTF_USE_CUSTOM_SMALLVECTOR=0 -DSIMDJSON_THREADS_ENABLED=1 -I/Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/include -isystem /Users/vagrant/Data/installed/x64-osx/include -fPIC -g -std=gnu++17 -arch x86_64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk -MD -MT CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o -MF CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o.d -o CMakeFiles/fastgltf.dir/src/fastgltf.cpp.o -c /Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/src/fastgltf.cpp
In file included from /Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/src/fastgltf.cpp:55:
In file included from /Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/include/fastgltf/parser.hpp:32:
/Users/vagrant/Data/buildtrees/fastgltf/src/v0.6.0-ea6ba6bf5e.clean/include/fastgltf/types.hpp:34:10: fatal error: 'memory_resource' file not found
#include <memory_resource>
         ^~~~~~~~~~~~~~~~~
1 error generated.
```

--- a/ports/fastgltf/portfile.cmake
+++ b/ports/fastgltf/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO spnda/fastgltf
     REF "v${VERSION}"
-    SHA512 85b946f9ea849bcbbb77ff5d4dc8196d3348757cf6a940be1a50923158a31aa7b43aebed2799256cb3d303a81fa28e5eaeb000b6ecca3ab15f6a7a20908d8e8f
+    SHA512 6cda7e50d7fe01428e0a03d3f590fe7b680bfa4b6fcdbd1c6a118ac01c925099e63b34380b053adc323e2aaaaead42bda450d1eaf66b60af6ad2aafb68828d01
     HEAD_REF main
     PATCHES find_package.patch
 )

--- a/ports/fastgltf/vcpkg.json
+++ b/ports/fastgltf/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "fastgltf",
-  "version": "0.5.0",
+  "version": "0.6.1",
   "port-version": 1,
   "description": "Blazing fast C++17 glTF 2.0 loader powered by SIMD",
   "homepage": "https://github.com/spnda/fastgltf",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2541,7 +2541,7 @@
       "port-version": 4
     },
     "fastgltf": {
-      "baseline": "0.5.0",
+      "baseline": "0.6.1",
       "port-version": 1
     },
     "fastio": {

--- a/versions/f-/fastgltf.json
+++ b/versions/f-/fastgltf.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b098e0b272a75e2c12b898a54b6b83a49f981599",
+      "version": "0.6.1",
+      "port-version": 1
+    },
+    {
       "git-tree": "28cc725ac91ea72117083d152661ba131f8bca94",
       "version": "0.5.0",
       "port-version": 1

